### ISSUE TITLE
How long is long?

### ISSUE THREAD

**Issue by gevezex** [AndriyMulyar/bert_document_classification#12]
Is there a max token length for this long document? I mean if a document has 10k words, would it still work?

**Comment by AndriyMulyar** [comment]
Yes - but it will of course use more memory. I'd suggest to try finetuning a pretrained longformer if your documents truly are that long.
